components/layout_2020/flow/text_run.rs

```diff
@@ -442,7 +439,7 @@ impl TextRun {
             return false;
         }
 
-        let last_byte = self.text.as_bytes().get(run.range.end().to_usize() - 1);
+        let last_byte = self.text.as_bytes().get(run.range.begin().to_usize());
         last_byte == Some(&b'\n')
```

Comment:
Maybe "last" doesn't make as much sense any longer:

```suggestion
        let byte = self.text.as_bytes().get(run.range.begin().to_usize());
        byte == Some(&b'\n')
```